rust/src/lib/ifaces/ipsec.rs

```diff
@@ -90,6 +90,8 @@ pub struct LibreswanConfig {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub rightrsasigkey: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
+    pub rightcert: Option<String>,
```

Comment:
Maybe also consider updating the documentation in line 20 to add the rightcert ? 

```
/// interfaces:
/// - name: hosta_conn
///   type: ipsec
///   ipv4:
///     enabled: true
///     dhcp: true
///   libreswan:
///     right: 192.0.2.252
///     rightid: '@hostb.example.org'
///     left: 192.0.2.251
///     leftid: '%fromcert'
///     leftcert: hosta.example.org
///     ikev2: insist
/// ```
```

Reply:
Not sure about this. Maybe I'm wrong, but I think that using rightcert is not the most common config, so better to leave as example a more common one. Otherwise users might think that they need to add it when they don't.

Reply:
Using rightcert is not common. The cert should match with rightid or IP address.